src/test/java/tr/edu/ogu/ceng/service/StudentTest.java

```diff
@@ -52,7 +52,7 @@ void should_save_one_student() {
 
 		when(studentRepository.save(any(Student.class))).thenReturn(studentToSave);
 
-		final var actual = studentService.addStudent(new Student());
+		var actual = studentService.addStudent(new Student());
```

Comment:
Bu kısımda da boş constructor ile Student verilmiş.

Reply:
CompanyTest in onaylanmasının ardından student içinde formatı düzelteceğim.